app/src/main/res/layout/survey_item.xml

```diff
@@ -1,33 +1,24 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:minHeight="?android:attr/listPreferredItemHeight"
-    android:gravity="center_vertical"
-    android:padding="8dp" >
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                xmlns:tools="http://schemas.android.com/tools"
+                android:minHeight="?android:attr/listPreferredItemHeight"
+                android:gravity="center_vertical"
+                android:padding="8dp">
 
     <TextView
-        android:id="@+id/text1"
+        android:id="@+id/survey_name_tv"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:textSize="22sp" />
-
-    <TextView
-        android:id="@+id/text2"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginLeft="8dp"
-        android:layout_marginStart="8dp"
-        android:layout_toRightOf="@id/text1"
-        android:layout_toEndOf="@id/text1"
-        android:layout_alignBaseline="@+id/text1"
-        android:enabled="false"
-        android:textSize="18sp" />
+        android:textColor="@color/text_color_primary"
+        tools:text="Farmer Rice Diary: Pest - Disease"
```

Comment:
@muloem tools prefix allows you to display a text only in the xml editor (layout preview), the users will never see it. more info here: https://developer.android.com/studio/write/tool-attributes.html check the tools instead of android section.